internal/events/aggregator.go

```diff
@@ -618,10 +622,6 @@ func (ag *aggregator) readyForDispatch(ctx context.Context, msg *core.Message, d
 		var handlerResult definitions.HandlerResult
 		handlerResult, err = ag.definitions.HandleDefinitionBroadcast(ctx, &state.BatchState, msg, data, tx)
 		log.L(ctx).Infof("Result of definition broadcast '%s' [%s]: %s", msg.Header.Tag, msg.Header.ID, handlerResult.Action)
-		if handlerResult.Action == core.ActionReject {
-			log.L(ctx).Infof("Definition broadcast '%s' rejected: %s", msg.Header.ID, err)
-			err = nil
```

Comment:
Note that this block should now fall through and return the error to its caller (line 528), which will then log the error and add it to the message, but should not cause retries. I'm trying to be very careful not to reintroduce the bugs fixed in https://github.com/hyperledger/firefly/pull/1278.